registry/handlers/layer.go

```diff
@@ -49,8 +49,8 @@ type layerHandler struct {
 // response.
 func (lh *layerHandler) GetLayer(w http.ResponseWriter, r *http.Request) {
 	ctxu.GetLogger(lh).Debug("GetImageLayer")
-	layers := lh.Repository.Layers()
-	layer, err := layers.Fetch(lh.Digest)
+	layerStore := lh.Repository.Layers()
+	layerReader, err := layerStore.Fetch(lh.Digest)
```

Comment:
Just updated these lines for clarity. "layers" and "layer" gave me no idea what the types were and there are quite a few structs with layer in the name that might have been candidates.